static/app/utils/issueTypeConfig/uptimeConfig.tsx

```diff
@@ -6,7 +6,7 @@ const uptimeConfig: IssueCategoryConfigMapping = {
   _categoryDefaults: {
     actions: {
       archiveUntilOccurrence: {enabled: true},
-      delete: {enabled: false},
+      delete: {enabled: true},
```

Comment:
have we checked with the uptime team that uptime issues can be deleted? i might be wrong but i thought there was talk about not being able to delete these at one point

Reply:
Thanks @roggenkemper I will follow up.